crypto/kms/pubkeystore.go

```diff
@@ -45,23 +45,26 @@ const (
 
 var (
 	// pks holds the singleton instance
-	pks     *PublicKeyStore
-	pksOnce sync.Once
+	pks *PublicKeyStore
+	// for easy test we make PksOnce exported
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var PksOnce should be of the form "PksOnce ..."